src/main.js

```diff
@@ -9,12 +9,14 @@ import store from './store'
 import FirebaseAuth from '@/utils/firebase.js'
 import router from './router'
 import SvgIcon from '@/components/SvgIcon'
+import SyncButton from '@/components/SyncButton'
 import '@/assets/scss/main.scss'
 
 const requireAll = requireContext => requireContext.keys().map(requireContext)
 const req = require.context('@/assets/icons', true, /\.svg$/)
 requireAll(req)
 Vue.component('icon', SvgIcon)
+Vue.component('SyncButton', SyncButton)
```

Comment:
Why `SyncButton` should be globally registered? Can't we import it when we need it?

Reply:
I'd making it more like BootstrapVue, where components are globally visible

Reply:
Any thoughts?

Reply:
That's fair enough, but I think importing the component when we need it would be better. It would shrink down the chunk size after the build.